app/routes/polls/components/PollEditor.tsx

```diff
@@ -22,17 +23,15 @@ import type { ReactNode } from 'react';
 import type {
   fieldArrayMetaPropTypes,
   fieldArrayFieldsPropTypes,
-} from 'redux-form';
+} from 'react-final-form';
```

Comment:
`react-final-form` does not export these types. However, I don't seem to find them in `redux-form` neither ... Has this ever worked?

Reply:
Probably not 😕

Reply:
Probably existed in the flow type definitions